app/views.py

```diff
@@ -43,6 +43,8 @@ def results(request):
         blank_form = QueryForm()
         data_to_render = {'error_message': None, 'form': blank_form}
 
+        movie = OMDbAPI().search(search_term)
```

Comment:
Is it necessary to hit OMDb on every request? It seems to me that this data could be more cheaply acquired from the local database rather than issuing a web call